source/renderer/app/components/wallet/skins/AmountInputSkin.scss

```diff
@@ -9,7 +9,7 @@
   font-size: 14px;
   position: absolute;
   right: 19px;
-  user-select: text;
+  user-select: none;
```

Comment:
@DeeJayElly why did we add this?

Reply:
Text should be non selectable according to -> https://docs.google.com/document/d/1VuXLJ-IysCXUuMUkEHFABCCKAMo0OCJEzVIwLoCbKgs/edit?disco=AAAAJP3XFi0

Reply:
Cool!